static/sass/_pattern_careers-modal.scss

```diff
@@ -23,9 +23,15 @@
       border-bottom: 0;
       display: block;
       margin-bottom: $spv--medium;
+      padding-top: 0;
+
+      &::after {
+        height: 0;
+      }
 
       .p-modal__close {
         margin-left: -$spv--large;
+        margin-top: $spv--medium;;
```

Comment:
Same here, could affect other existing modals?

Reply:
Same as above but removed the double semi colons